templates/zerver/api/changelog.md

```diff
@@ -20,6 +20,12 @@ format used by the Zulip server that they are interacting with.
 
 ## Changes in Zulip 6.0
 
+**Feature level 125**
+
+* [`POST /register`]: Added `display_emoji_reaction_users` as a display
+  setting which shows the names of reacting users on a message when total
+  reactions are 3 or less.
```

Comment:
Rewrote this description, most importantly listing all 4 affected endpoints. (I just copy-pasted from the last new display setting).